Dockerfile

```diff
@@ -1,23 +1,14 @@
-ARG DOCKER_ARCH=amd64
-
```

Comment:
I deleted all the `ARCH` stuff here to make the dockerfile easier to build -- it now depend on the `--platform` argument to `docker build` instead of specifying the architecture as an `--arg`